ibmdbpy/base.py

```diff
@@ -1343,8 +1338,9 @@ def append(self, idadf, df, maxnrow=None):
         """
         # SANITY CHECK : maxnrow
         if maxnrow is None:
-            # Note : 8000 is an empirical maximum number of cells
-            maxnrow = int(8000 / len(df.columns))
+            # Note : previously, int(8000 / len(df.columns)) was an empirical value for the maximum number of cells
```

Comment:
The previous value can easily be found via git, it's not worth keeping around. If you really want to improve this comment, tell the reader how you've measured that 10000 is better than 8000.

Reply:
I have changed it to 

> # Note : it has been measured on a big dataset (>1 million rows) that int(100000 / len(df.columns)) performs better than previous empirical value int(8000 / len(df.columns))